### DISCUSSION TITLE
SMTP SUPPORT

### DISCUSSION THREAD

**Discussion by roccinio** [inspireui/support#5498] [Working on Features]
Hello,
to my surprise i saw that you do not support smtp.
This is very problematic since the whole concept of a successful eshop is to have your emails delivered and not end up in the spam folder.
By not supporting this (https://wpmailsmtp.com/ )you are liming us from all 3rd party mail delivery systems like Elastic mail, Mailjet etc.

Regards

**Comment by minhcasi** [comment]
SMTP should be config from your WordPress admin side, you could use any STMP plugin - thanks
